### ISSUE TITLE
Cannot insert new video track below existing video tracks

### ISSUE THREAD

**Issue by ochilan** [mltframework/shotcut#414]
Maybe there is a way to accomplish this but I haven't found it yet. I sometimes need to insert a "background" track below all existing video tracks, but I cannot find a way to insert one at that position directly. Since there is no way to reorder tracks yet (I know, it's on the roadmap), this necessitates a rather cumbersome rearranging of existing clips. Maybe some other method to insert a track at an arbitrary position could be devised?

**Comment by SleekPanther** [comment]
Unfortunately it seems like this is currently impossible. I really hope reordering tracks is introduced

**Comment by ddennedy** [comment]
feature request, not bug

**Comment by anderbubble** [comment]
I don't see this as a feature request, as the render z-order of tracks directly contradicts the order of clips in the timeline. What's the point of inserting a track in the middle of the timeline if not to affect how the tracks overlay atop each other?

imo, this is a bug; and either the ability to insert a track at a relatively arbitrary position should be removed to not be misleading, or the render order should reflect the order the tracks appear in the timeline; not the order that they were added to it regardless of their position.

I've got a minimal example here demonstrating how I encountered this behavior:

https://forum.shotcut.org/t/track-render-order-follows-creation-not-timeline-order/10152
